### PR TITLE
Swaggo

### PR DESCRIPTION
**Pull request by pzabolotniy**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -15,11 +15,16 @@ type Service struct {
 	Address string `mapstructure:"address"`
 }
 
+type Server struct {
+	ShutdownTimeout int `mapstructure:"shutdown_timeout"` // in minutes
```
Comment (pzabolotniy) [inline internal/config/config.go]: use native time.Duration type and time.Duration-formatted values in the config

File: main.go [hunk 2]
```diff
@@ -14,11 +14,15 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 
+	"github.com/Sugar-pack/rest-server/docs"
+	_ "github.com/Sugar-pack/rest-server/docs"
```
Comment (pzabolotniy) [inline main.go]: duplicate import

File: internal/config/config.go (hunk 1 above)
Comment (Sugar-pack) [reply internal/config/config.go]: done

File: main.go (hunk 2 above)
Comment (Sugar-pack) [reply main.go]: done

**Pull request merged**